### ISSUE TITLE
About the comparison experiment

### ISSUE THREAD

**Issue by devyao** [AlibabaResearch/efficientteacher#72]
Hello, are there any experiments supervised yolov5 that use the training set and validation set, and the semi-supervised experiments efficientteacher that use the training set, validation set and unlabeled data? Thank you for your work！
